shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -139,6 +143,11 @@ public void deleteBridgeExecutor(ProcessorDTO processorDTO) {
         if (!bridgeDeleted) {
             // TODO: we might need to review this use case and have a manager to look at a queue of objects not deleted and investigate. Unfortunately the API does not give us a reason.
             LOGGER.warn("BridgeExecutor '{}' not deleted", processorDTO);
+            LOGGER.debug("BridgeExecutor '{}' was not found. Notifying manager that it has been deleted.", processorDTO.getId());
```

Comment:
Duplicated log? And actually the statement in the `LOGGER.debug` is not (always) true

Reply:
See https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/620